force_wfmanager/wfmanager_task.py

```diff
@@ -54,6 +57,9 @@ class WfManagerTask(Task):
     #: ZeroMQ Server to receive information from the running BDSS
     _zmq_server = Instance(ZMQServer)
 
+    #: False when the ui must be disabled because of a running computation
+    _ui_enabled = Bool(True)
```

Comment:
actually, it's probably better to have a meaningful flag here. "computation_running". Because some things are actually enabled and can stay enabled while the evaluation is running, like the plot, or some menus (help)